src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,13 +188,34 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resourse-dropdown [r language counts]
```

Comment:
Since this is used as an `:item-label`, perhaps a name like `resource-label`? Also, it could be private.